boards/apollo3/lora_things_plus/.cargo/config.toml

```diff
@@ -1,6 +1,14 @@
 # Licensed under the Apache License, Version 2.0 or the MIT License.
 # SPDX-License-Identifier: Apache-2.0 OR MIT
-# Copyright Tock Contributors 2022.
+# Copyright Tock Contributors 2024.
 
-[target.'cfg(target_arch = "arm")']
-runner = "./run.sh"
```

Comment:
What happened to this?

Reply:
Got lost when I moved the `apollo3/.cargo/config.toml` to each board. Fixed